docs/content/guides/dagster/using-environment-variables-and-secrets.mdx

```diff
@@ -102,7 +102,23 @@ Refer to the [Dagster Cloud Branch Deployments example](#example-2-dagster-cloud
 
 [Configurable Dagster objects](/concepts/configuration/config-schema) - such as ops, assets, resources, I/O managers, and so on - can accept configuration from environment variables.
 
-To access an environment variable as part of a Dagster configuration, use the following syntax:
+#### In Python code
+
+To access an environment variable as part of a Dagster configuration in Python code, use the following syntax:
```

Comment:
user had [this feedback](https://dagster.slack.com/archives/C01U5LFUZJS/p1681313419629829) in slack:
> I've seen env variables expressed as EnvVar() and os.getenv in different parts of the documentation. It's not clear to me which one we should use with Pythonic resources or why EnvVar exists in the first place

so maybe worth explaining the benefit of using `EnvVar`

Reply:
* Observability and documentation. You can understand where values are coming from while not displaying potentially sensitive information in the UI.
* Testability. It is very straightforward to provide a raw string value instead of an environment variable
* (In cloud) The ability to edit environment variables directly in the UI

If none of these value propositions is interesting, this is all plain python, and the user should feel free to invoke `os.getenv` directly.

Reply:
Updated with some more explanation of Dagster's env var system